addons/repair/migrations/13.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+_copy_columns = {
+    "repair_fee": [('invoice_line_id', None, None)],
```

Comment:
You can't copy columns. You have to rename them. If not, wrong constraints will be kept.